Resources/Private/Partials/Lists/Pagination.html

```diff
@@ -18,11 +18,14 @@
             </li>
         </f:else>
     </f:if>
-    <f:if condition="{pagination.hasLessPages}">
-        <li>…</li>
-    </f:if>
-    <f:comment>add pages between first and last page</f:comment>
-    <f:for each="{pagination.pagesG}" as="page">
+    <f:comment>is no longer needed, is provided in pagination.pagesR</f:comment>
+    <f:comment>
+        <f:if condition="{pagination.hasLessPages}">
+            <li>…</li>
+        </f:if>
+    </f:comment>
```

Comment:
If it is no longer needed (and working), please remove it entirely.

Reply:
done